tests/cypress/integration/coming-soon-woo.cy.js

```diff
@@ -46,7 +47,7 @@ describe( 'Coming Soon with WooCommerce', { testIsolation: true }, () => {
 		cy.get(
 			'#wp-toolbar #wp-admin-bar-woocommerce-site-visibility-badge a.ab-item'
 		)
-			.contains( 'a', 'Coming soon' )
+			.contains( 'a', 'oming soon' )
```

Comment:
Making sure this is not a typo? @circlecube

Reply:
Yea, sometimes it's "Coming soon" and sometimes "Store coming soon" - that's why it was failing. There's a setting in woo to only set store pages to coming soon instead of the whole site.